### PR TITLE
WD-11736 build /pro/devices page

### PR DESCRIPTION
**Pull request by minkyngkm**
## Done

- Build /pro/devices page and its contact form 

## QA

- Go to /pro/devices
- Compare page against [copy doc](https://docs.google.com/document/d/1i_3nPkSpCQJ4jfOwN380sdfuTxaiDuDIgDfk01wVPqo/edit)
- Check CTA button for the form

## Issue / Card

Fixes #https://warthogs.atlassian.net/browse/WD-11736

### CODE REVIEW

**Comment by anthonydillon** [comment]
@lyubomir-popov we are missing a few scaled logo's here if you could help please?

**Comment by minkyngkm** [comment]
@lyubomir-popov I made a comment in the [copy doc](https://docs.google.com/document/d/1i_3nPkSpCQJ4jfOwN380sdfuTxaiDuDIgDfk01wVPqo/edit?disco=AAABORyEm1g) for the logos and this is [the devices page](https://ubuntu-com-14028.demos.haus/pro/devices) I built, for your reference 🙏

**Comment by minkyngkm** [comment]
@leiaru-c the demo for the page is [here](https://ubuntu-com-14028.demos.haus/pro/devices) 🙏

File: static/sass/_pattern_lists.scss [hunk 1]
```diff
@@ -312,7 +312,8 @@
 }
 
 @mixin ubuntu-p-list-ticked {
-  .p-list--ticked {
+  .p-list--ticked,
+  .p-list--ticked--small {
```
Comment (anthonydillon) [inline static/sass/_pattern_lists.scss:316]: Is this a new component? Can you raise an issue on Vanilla and comment it here please?

File: templates/pro/devices.html [hunk 2]
```diff
@@ -0,0 +1,529 @@
...
+      <div class="row">
+        <div class="col-medium-6 col-9 col-start-large-4">
+          <h1>Ubuntu Pro for Devices</h1>
+          <p>
+            Pay a one-time-fee per device and get 10 years of security maintenance. Ideal to comply with the Cyber Resilience Act and other regulations.
+          </p>
+          <a href="/pro/devices#get-in-touch" class="p-button--positive">Get in touch to learn more</a>
+        </div>
+      </div>
+      <div class="p-suru u-no-padding--top"></div>
```
Comment (anthonydillon) [inline templates/pro/devices.html:31]: You can wrap the containing `p-strip` with `p-suru` to add the decorator to the end of the strip.

File: templates/pro/devices.html [hunk 3]
```diff
@@ -0,0 +1,529 @@
...
+        </div>
+      </div>
+      <div class="row--50-50">
+        <div class="col u-vertically-center">
+          <div class="u-embedded-media">
+            <iframe class="u-embedded-media__element"
+                    width="560"
+                    height="315"
+                    src="https://www.youtube.com/embed/tNECRG5r2sg"
+                    title="YouTube video player"
```
Comment (anthonydillon) [inline templates/pro/devices.html:64]: Can you give this a better title please?

File: templates/pro/devices.html [hunk 4]
```diff
@@ -0,0 +1,529 @@
...
+            {{ image (
+            url="https://assets.ubuntu.com/v1/dbd97444-partner-with-us.svg",
+            alt="",
+            width="204",
+            height="150",
+            hi_def=True,
+            loading="lazy",
+            ) | safe
+            }}
+          </div>
```
Comment (anthonydillon) [inline templates/pro/devices.html:267]: Looks like this should go below the heading?

File: templates/pro/devices.html [hunk 5]
```diff
@@ -0,0 +1,529 @@
...
+      </div>
+    </section>
+    <div class="row">
+      <hr class="p-rule is-muted" />
+    </div>
+    <div class="p-strip is-deep">
+      <div class="row u-vertically-center">
+        <div class="col-10">
+          <h2>
+            <a href="">Placeholder for IoT Security whitepaper&nbsp;&rsaquo;</a>
```
Comment (anthonydillon) [inline templates/pro/devices.html:366]: We should probably remove this incase it gets merged.

File: templates/pro/devices.html [hunk 6]
```diff
@@ -0,0 +1,529 @@
...
+              <p>Check out the press release to discover more about Pro for Devices and our launch partners.</p>
+            </div>
+            <div>
+              <a href="/engage/software-defined-iot">Software defined IoT</a>
+              <p>
+                On average, organisations developing IoT products will spend 50% more resources on software than hardware. Read our whitepaper to get ahead of costs and design your device, software first.
+              </p>
+            </div>
+            <div>
+              <a href="/engage/cto-guide-real-time-kernel#:~:text=A%20real%2Dtime%20system%20ensures,workloads%20on%20real%2Dtime%20systems.">Introduction to real-time kernel</a>
```
Comment (anthonydillon) [inline templates/pro/devices.html:505]: Probably don't need the test selector on the link

File: templates/templates/_footer-copyright-and-legal.html [hunk 7]
```diff
@@ -9,7 +9,7 @@
       <a class="p-link--soft" accesskey="9" href="/legal/data-privacy"><small>Data privacy</small></a>
     </li>
     <li class="p-inline-list__item">
-      <a href="" class="p-link--soft js-revoke-cookie-manager"><small>Manage your tracker settings</small></a>
+      <a class="p-link--soft js-revoke-cookie-manager"><small>Manage your tracker settings</small></a>
```
Comment (anthonydillon) [inline templates/templates/_footer-copyright-and-legal.html:12]: This is invalid HTML

File: templates/pro/devices.html [hunk 8]
```diff
@@ -0,0 +1,529 @@
...
+        <div class="p-logo-section--dense">
+          <div class="p-logo-section__items">
+            <div class="p-logo-section__item">
+              {{ image (
+              url="https://assets.ubuntu.com/v1/c1b1d4a9-ros-logo.png",
+              alt="",
+              width="288",
+              height="288",
+              hi_def=True,
+              loading="auto|lazy"
```
Comment (anthonydillon) [inline templates/pro/devices.html:146]:
```suggestion
              loading="lazy"
```

File: templates/pro/devices.html [hunk 9]
```diff
@@ -0,0 +1,529 @@
...
+              }}
+            </div>
+            <div class="p-logo-section__item">
+              {{ image (
+              url="https://assets.ubuntu.com/v1/56d8741e-node-logo.png",
+              alt="",
+              width="196",
+              height="253",
+              hi_def=True,
+              loading="auto|lazy"
```
Comment (anthonydillon) [inline templates/pro/devices.html:179]:
```suggestion
              loading="lazy"
```

File: templates/pro/devices.html [hunk 10]
```diff
@@ -0,0 +1,529 @@
...
+              width="92",
+              height="100",
+              hi_def=True,
+              loading="lazy"
+              ) | safe
+              }}
+            </div>
+            <div class="p-logo-section__item">
+              {{ image (
+              url="https://assets.ubuntu.com/v1/fa78f5b4-openjdk logo.png",
```
Comment (anthonydillon) [inline templates/pro/devices.html:163]:
```suggestion
              url="https://assets.ubuntu.com/v1/fa78f5b4-openjdk%20logo.png",
```

File: templates/pro/devices.html [hunk 11]
```diff
@@ -0,0 +1,529 @@
...
+              width="196",
+              height="253",
+              hi_def=True,
+              loading="auto|lazy"
+              ) | safe
+              }}
+            </div>
+            <div class="p-logo-section__item">
+              {{ image (
+              url="https://assets.ubuntu.com/v1/5dbf023e-docker logo.png",
```
Comment (anthonydillon) [inline templates/pro/devices.html:185]:
```suggestion
              url="https://assets.ubuntu.com/v1/5dbf023e-docker%20logo.png",
```

File: templates/pro/devices.html [hunk 12]
```diff
@@ -0,0 +1,529 @@
...
+      <hr class="p-rule is-muted" />
+    </div>
+    <section class="p-section">
+      <div class="u-fixed-width">
+        <h2 class="u-sv2">Backed by leading ODMs</h2>
+      </div>
+      <div class="row--50-50">
+        <div class="col u-hide--medium u-hide--small">
+          {{ image (
+          url="https://assets.ubuntu.com/v1/80342764-aaeon-logo @372px.png",
```
Comment (anthonydillon) [inline templates/pro/devices.html:381]:
```suggestion
          url="https://assets.ubuntu.com/v1/80342764-aaeon-logo%20@372px.png",
```

File: templates/pro/devices.html [hunk 13]
```diff
@@ -0,0 +1,529 @@
...
+          </blockquote>
+        </div>
+      </div>
+      <div class="row">
+        <hr class="p-rule is-muted" />
+      </div>
+      <div class="row--50-50">
+        <div class="col u-hide--medium u-hide--small">
+          {{ image (
+          url="https://assets.ubuntu.com/v1/acab2168-advantech-logo @372px.png",
```
Comment (anthonydillon) [inline templates/pro/devices.html:405]:
```suggestion
          url="https://assets.ubuntu.com/v1/acab2168-advantech-logo%20@372px.png",
```

File: templates/pro/devices.html [hunk 14]
```diff
@@ -0,0 +1,529 @@
...
+          </blockquote>
+        </div>
+      </div>
+      <div class="row">
+        <hr class="p-rule is-muted" />
+      </div>
+      <div class="row--50-50">
+        <div class="col u-hide--medium u-hide--small">
+          {{ image (
+          url="https://assets.ubuntu.com/v1/2dea272c-dfi-logo @372px.png",
```
Comment (anthonydillon) [inline templates/pro/devices.html:429]:
```suggestion
          url="https://assets.ubuntu.com/v1/2dea272c-dfi-logo%20@372px.png",
```

File: templates/pro/devices.html [hunk 15]
```diff
@@ -0,0 +1,529 @@
...
+              </p>
+            </div>
+            <div>
+              <a href="/engage/guide-to-deploying-ai-at-the-edge">Discover AI at the Edge</a>
+              <p>
+                AI is breaking out of the data centre and moving to the edge. From self-driving cars to industrial assembly lines, the ability to deploy machine learning models on embedded devices and generate real-time, AI-driven insight out in the field offers tremendous potential.
+              </p>
+            </div>
+          </div>
+          <button href="/pro/devices#get-in-touch"
```
Comment (anthonydillon) [inline templates/pro/devices.html:517]: This needs to be a link to have a `href`

**Review by anthonydillon — CHANGES_REQUESTED** [review]
Nice work @minkyngkm, Few comments and suggestions in the code.

**Comment by leiaru-c** [comment]
Nice work! I have a couple of suggestions:

1/ Heading:
It seems a bit off in terms of typography balance. Would it be possible to use the same layout that the /pro homepage on the header so it looks like this:
image

2/ "For secure and compliant IoT device deployment" section:
I think the current setup feels a bit cluttered. I would suggest to try a layout like this one: 
image

3/ Logo section:
As mentioned in standup, some of these logos are too big and would need to be scaled down. Try to aim for something similar to this size (as seen on/pro): 
image

4/ "Develop at speed with the Ubuntu ecosystem" section:
 These icons are getting phased out, so maybe try this layout: 
image

5/ "Meet your compliance requirements faster" section:
Same thing as before, I think removing the shields icon would be better.

6/ "Device management included" section:
The layout is not consistent with other similar sections, I would suggest trying something like:
image
This would make it closer to what we have on /pro.

7/"Backed by leading ODMs" section:
As is, this section takes a big space in the page and is not very consistent with what we have on /pro. Try to use this layout instead: 
image
Replace the "used by pro" in the screenshot with the "Backed by leading ODMs" section title and it should follow the same logic :)

8/"Resources" section:
This section is a bit difficult to parse. We can look to what was done on /pro for inspiration. I would suggest to try doing something like this:
image

so it's similar to this: 
image

In this case, the links for each section should be converted in this link CTA format with the line under the sections to draw the eye on the different options :)

8/Contact us:
Finally on this section, I would actually make it part of the previous section like this:
image

File: templates/pro/devices.html [hunk 16]
```diff
@@ -0,0 +1,538 @@
...
+              </p>
+            </div>
+          </div>
+          <hr class="p-rule is-fixed-width is-muted" />
+          <div class="row">
+            <div class="col-3">
+              <div class="u-pro-logo">
+                {{ image (
+                  url="https://assets.ubuntu.com/v1/2dea272c-dfi-logo%20@372px.png",
+                  alt="",
```
Comment (anthonydillon) [inline templates/pro/devices.html:416]: We should put the component name in the alt here.

File: templates/pro/devices.html (hunk 11 above)
Comment (anthonydillon) [reply templates/pro/devices.html:185]: Is there a reason you didn't add these changes?

**Review by anthonydillon — APPROVED** [review]
Code looks good with a few small suggests and questions

**Comment by anthonydillon** [comment]
QA is ok but logos need to be fixed

**Pull request merged**